### PR TITLE
update docs links

### PR DESCRIPTION
**Pull request by Arhell**
resolve https://github.com/fluxcd/website/issues/899

### CODE REVIEW

**Comment by Arhell** [comment]
cc @dholbach

**Review by dholbach — COMMENTED** [review]
Maybe you can apply these changes?

File: content/en/blog/2022-04-04-march-update/index.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ here's a quick list of our highlights:
   providing explicit status conditions which conform to the
   Kubernetes `kstatus` conventions.
 - A new annotation
-  ([kustomize.toolkit.fluxcd.io/ssa](http://kustomize.toolkit.fluxcd.io/ssa):
+  ([kustomize.toolkit.fluxcd.io/ssa](https://kustomize.toolkit.fluxcd.io/ssa):
```
[the remark this answers is not in the document]
Comment (dholbach) [reply content/en/blog/2022-04-04-march-update/index.md:52]: Maybe just place in ` instead?

File: content/en/blog/2021-10-29-november-update/index.md [hunk 2]
```diff
@@ -263,7 +263,7 @@ styling, beautification and bringing in a nicer 404 page! 🤩
 On top of that: more frequently asked questions and many more updates to
 the docs, particularly around the new world order involving Server-Side
 Apply! We updated [our Contributor
-docs](http://fluxcd.io/contributing/) as well! Hope to see
+docs](https://fluxcd.io/contributing/) as well! Hope to see
```
[the remark this answers is not in the document]
Comment (Arhell) [reply content/en/blog/2021-10-29-november-update/index.md]: done

File: content/en/blog/2021-11-30-december-update/index.md [hunk 3]
```diff
@@ -234,7 +234,7 @@ January 6) -- note: there will be no meeting on December 23rd due to a
 company-wide holiday shutdown. Find the Zoom link on the [CNCF Flux Dev
 calendar](https://lists.cncf.io/g/cncf-flux-dev/calendar),
 (subscribe), or locate the link directly on the [Flux main
-page](http://fluxcd.io/#calendar).
+page](https://fluxcd.io/#calendar).
```
[the remark this answers is not in the document]
Comment (Arhell) [reply content/en/blog/2021-11-30-december-update/index.md]: done

File: content/en/blog/2021-11-30-december-update/index.md [hunk 4]
```diff
@@ -208,7 +208,7 @@ based on top of the most recent versions of Flux.
 The Flux Bug Scrub in the last week of November (25th) coincided with
 the US Thanksgiving holiday, so it was not held. Each weekly Bug Scrub
 can be found (and linked Zoom details) on the
-[fluxcd.io/\#calendar](http://fluxcd.io/#calendar)
+[fluxcd.io/\#calendar](https://fluxcd.io/#calendar)
```
[the remark this answers is not in the document]
Comment (Arhell) [reply content/en/blog/2021-11-30-december-update/index.md]: done

File: content/en/blog/2022-04-04-march-update/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Arhell) [reply content/en/blog/2022-04-04-march-update/index.md]: roll back

**Review by dholbach — APPROVED** [review]
Thanks a lot - this is super!

**Pull request merged**